src/containers/Marketplace/GenerativeListings.tsx

```diff
@@ -45,15 +47,49 @@ function sortValueToSortVariable(val: string) {
 
 interface Props {
   token: GenerativeToken
+  urlQuery: Record<string, string>
```

Comment:
We only need to provide the actualy initial sorting to the component

Reply:
I made this change and please review my PR again.

Reply:
What I meant was not to serialize the query object but only the value what we need. we dont have any other query params for now so we can literally return the sorting value. 

rename sort -> sorting